openapi/spec.go

```diff
@@ -194,6 +194,7 @@ type Operation struct {
 	Deprecated   bool              `json:"deprecated,omitempty" yaml:"deprecated,omitempty"`
 	Servers      []*Server         `json:"servers,omitempty" yaml:"servers,omitempty"`
 	XCodeSamples []*XCodeSample    `json:"x-codeSamples,omitempty" yaml:"x-codeSamples,omitempty"`
+	XInternal    bool              `json:"x-internal,omitempty" yaml:"x-internal,omitempty"`
```

Comment:
What does the spec say about it ? Is it preferable if its omitted when its `false` (external), or should it be present for clarity and/or compatibility with the rendering interfaces ?

Reply:
Hmm. Can't find anything really 🙈  Maybe it's a bit unclear as this is an extension, so I'm not sure it's covered in the spec?

Given that this is an extension, I'm leaning towards not showing `x-internal: false` (which should be the current behaviour) to not "pollute" the spec with default values of extensions. WDYT?

Reply:
Agreed, lets keep the `omitempty` flag then.